src/scripts/install-sops.sh

```diff
@@ -5,7 +5,7 @@ Install() {
   if ! [ -x "$(command -v ${TOOL_NAME})" ]; then
     mkdir -p "${BIN_PATH}"
     if [ "${TOOL_VERSION}" = "latest" ]; then
-      TOOL_VERSION=$(curl -s https://api.github.com/repos/mozilla/sops/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+      TOOL_VERSION=$(curl -s https://api.github.com/repositories/40684033/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
what is this magic number? the code before looks more correct :-)

Reply:
I agree, that original URL looked better, but when you call it
```
 curl -s https://api.github.com/repos/mozilla/sops/releases/latest
{
  "message": "Moved Permanently",
  "url": "https://api.github.com/repositories/40684033/releases/latest",
  "documentation_url": "https://docs.github.com/v3/#http-redirects"
}
```

Reply:
just -L for follow?

Reply:
ah sops has different repo 
```suggestion
      TOOL_VERSION=$(curl -s https://api.github.com/repos/getsops/sops/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```